analysis/statistics/handling_cost.py

```diff
@@ -0,0 +1,11 @@
+from typing import Tuple
+
+from heuristic.classes import Solution
+
+
+def handling_cost(solution: Solution) -> Tuple[str, float]:
+    """
+    Returns instance's handling costs.
+    """
+    return "handling_cost", \
```

Comment:
If you wrap this in `("handling_cost", <expression)`, you don't need the line continuation `\`.